docs/conceptual/how-omnitrace-works.rst

```diff
@@ -0,0 +1,299 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+*******************
+How Omnitrace works
```

Comment:
This topic looks like nomenclature.
Perhaps this topic should be titled Omnitrace glossary
Do you need the nomenclature subhead?

Reply:
Just looking more closely now:
![image](https://github.com/user-attachments/assets/6521747a-4375-4fae-84e5-ed3eab7c4a10)

Perhaps Glossary is a single topic.
Tips is another topic that goes into How to
Data collection modes is a conceptual topic

These three things don't seem right under How Omnitrace works

Reply:
I refactored this into three pages.